.github/workflows/checks.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
     - name: Ensure latest install manifest
       run: |
         echo "" > dist/install.yaml
-        IMG=ghcr.io/maiqueb/kubevirt-ipam-claims:latest make build-installer
+        IMG=ghcr.io/kubevirt/ipam-controller:latest make build-installer
```

Comment:
shouldnt it be as this repo name ? `kubevirt/ipam-extensions`
i do understand it is according the controller name, just wonder if it is fine that it is different
also the publish manifest use the repo name iiuc
https://github.com/kubevirt/ipam-extensions/pull/28/files#diff-b7ac418a59622d832059f2e255c23ec25a8ff3bf10471923e299a80602ed3640R62

Reply:
continuing with this as discussed with Quique